docs/src/physics/coriolis_forces.md

```diff
@@ -2,23 +2,40 @@
 
 The Coriolis model controls the manifestation of the term $\bm{f} \times \bm{u}$ in the momentum equation.
 
-## The $f$-plane approximation
+## The traditional and non-traditional $f$-plane approximation
 
 Under an $f$-plane approximation[^3] the reference frame in which
-the momentum and tracer equations are are solved rotates at a constant rate around a
-vertical axis, such that
+the momentum and tracer equations are are solved rotates at a constant rate. 
+
+### The traditional $f$-plane approximation 
+
+In the *traditional* $f$-plane approximation, the coordinate system rotates around
+a vertical axis such that
 ```math
     \bm{f} = f \bm{\hat z}
 ```
 where $f$ is constant and determined by the user.
 
+### The non-traditional $f$-plane approximation 
+
+In the *non-traditional* $f$-plane approximation, the coordinate system rotates around
+an axis in the $y,z$-plane, such that
+```math
+    \bm{f} = f_y \bm{\hat y} + f_z \bm{\hat z}
+```
+where $f_y$ and $f_z$ are constant and determined by the user.
+
+
 [^3]: The $f$-plane approximation is used to model the effects of Earth's rotation on
-      anisotropic fluid motion in a plane tangent to the Earth's surface. In this case, $\bm{f}$ is
+      anisotropic fluid motion in a plane tangent to the Earth's surface. In this case, the projection of Earth's
+      rotation vector at latitude $\varphi$ and onto a coordinate system in which $x, y, z$ correspond to the 
+      directions east, north, and up is
       ```math
-          \bm{f} \approx \frac{4 \pi}{\text{day}} \sin \varphi \bm{\hat z} \, , $
+          \bm{f} \approx \frac{4 \pi}{\text{day}} \left ( \cos \varphi \bm{\hat y} + \sin \varphi \bm{\hat z} \right ) \, , $
       ```
-      where $\phi$ is latitude and the Earth's rotation rate is approximately $2 \pi / \text{day}$.
-      This approximation neglects the vertical component of Earth's rotation vector at $\varphi$.
+      where the Earth's rotation rate is approximately $2 \pi / \text{day}$.
+      The *traditional* $f$-plane approximation neglects the $y$-component of this projection, which is appropriate for 
+      fluid motions with small horizontal-to-vertical aspect ratios.
```

Comment:
I think this should read "large horizontal-to-vertical aspect ratios"

Reply:
hmm yes, common mistake from me...